src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelperTest.java

```diff
@@ -37,28 +37,16 @@
 @SpringBootTest
 public class ServiceBusHelperTest {
 
-    @Mock
-    private IQueueClient queueClient;
+    @Autowired private ObjectMapper objectMapper;
```

Comment:
ok soooo ... which way is the standard?
inline Annotations or separate line annotations.
I really dont mind but lets be consistent ?